docs/content/basic_usage.md

```diff
@@ -25,19 +25,25 @@ env = pistonball_v6.env()
 Environments are generally highly configurable via arguments at creation, i.e.:
 
 ``` python
-cooperative_pong.env(ball_speed=18, left_paddle_speed=25,
-right_paddle_speed=25, is_cake_paddle=True, max_cycles=900, bounce_randomness=False)
+from pettingzoo.butterfly import cooperative_pong_v5
+
+cooperative_pong_v5.env(ball_speed=18, left_paddle_speed=25,
+right_paddle_speed=25, cake_paddle=True, max_cycles=900, bounce_randomness=False)
 ```
 
 ## Interacting With Environments
 
 Environments can be interacted with using a similar interface to Gymnasium:
 
-``` python
+``` python notest
+from pettingzoo.butterfly import cooperative_pong_v5
+
+env = cooperative_pong_v5.env()
 env.reset()
 for agent in env.agent_iter():
     observation, reward, termination, truncation, info = env.last()
     action = env.action_space(agent).sample() # this is where you would insert your policy
+    #TODO ValueError: when an agent is dead, the only valid action is None
```

Comment:
It needs to have an if termination or truncation action = None part, go to the online docs page for cooperative pong and you can see an example run script, just copy the code from that to make it consistent.

Reply:
Yep that did the trick